### PR TITLE
Updated SAA Template

### LINKED ISSUE #223: New SAA workflow, documentation, and template
### Description of Problem:
The team has identified difficulties in creating Service Approval Accelerator documents. In order to streamline this work, we have discussed a revised workflow. The revised workflow needs to be clearly articulated via documentation in this repo, and the template should be adjusted to match.

Untitled Diagram drawio(1)

### Potential Solutions:

Someone will need to take the lead, with collaboration from the community, to create the necessary documentation and template.

### PR DESCRIPTION
**Pull request by eddie-knight**
Adjusted the SAA template to close #223.

Marking this as a draft pending feedback from the community.

### CODE REVIEW

**Comment by AdrianHammond** [comment]
@eddie-knight  - few comments on [#233](https://github.com/finos/compliant-financial-infrastructure/blob/b1d72ede0d13a74a1cd01fe1152ab6e143ab4140/templates/ServiceApprovalAcceleratorTemplate.md)

1. Would it make the template easier to read if you moved the [policy example](https://github.com/finos/compliant-financial-infrastructure/blob/b1d72ede0d13a74a1cd01fe1152ab6e143ab4140/templates/ServiceApprovalAcceleratorTemplate.md#1232-ensure-deployment-from-an-unauthorized-container-registry-is-denied) into the [intro section](https://github.com/finos/compliant-financial-infrastructure/blob/b1d72ede0d13a74a1cd01fe1152ab6e143ab4140/templates/ServiceApprovalAcceleratorTemplate.md#service-approval-accelerator-saa-template)?
2. When I looked at the template yesterday the benchmark link was broken, just checked and CIS site is down. I will check again later and add another comment. 
3. As a baseline would it be acceptable for a vendor to use their latest CIS benchmark e.g. Google using the GKE CIS Benchmark instead of the K8s benchmark
4. In the baseline would it be acceptable to just put a link to the CIS benchmark being used or are we expecting all the policies being included in the baseline to be documented in the service accelerator

**Comment by eddie-knight** [comment]
Thanks @AdrianHammond! I added an answer to your question in the notes on the document, and adjusted the formatting to hopefully highlight those notes more clearly.

I didn't move the policy example yet (seemed good where it is), but we can discuss that more if you think it'll help.

Let me know what you think!

File: templates/ServiceApprovalAcceleratorTemplate.md [hunk 1]
```diff
@@ -1,659 +1,70 @@
...
 
-This section is meant to provide an opinionated approach towards the
-implementation security controls by domain. Although the approaches may
-not be a fit for all use cases or complete for production use, they are
-meant to guide the reader towards current best practices and design
-considerations to achieve security control objectives.
+_**Template Notes:**_
 
-**Controls and Architectures**
+- _Text here may be ommitted, or optionally used to provide any supplemental information related to the baseline._
```
Comment (thinkl33t) [inline templates/ServiceApprovalAcceleratorTemplate.md:12]: omitted

File: templates/ServiceApprovalAcceleratorTemplate.md [hunk 2]
```diff
@@ -1,659 +1,70 @@
...
+- _We will aim to stick to CIS as much as possible, though the specific benchmark will be changed on a case-by-case basis._
+- _Multiple benchmarks may be linked if required for full compliance coverage._
 
-This table maps Security Domain to the corresponding controls and
-architectural best practices as documented in AWS’ public documentation, 
-white papers, and blog posts.
+The policies included in the following document(s) are required for compliance,
+unless otherwise noted in section 2: [Ammendments to the Baseline](#2-ammendments-to-the-baseline)
```
Comment (thinkl33t) [inline templates/ServiceApprovalAcceleratorTemplate.md:17]: amendments

Comment (eddie-knight) [reply templates/ServiceApprovalAcceleratorTemplate.md]: changed

File: templates/ServiceApprovalAcceleratorTemplate.md (hunk 1 above)
Comment (eddie-knight) [reply templates/ServiceApprovalAcceleratorTemplate.md]: corrected

**Comment by abdullahgarcia** [comment]
Let's move things forward.

@AdrianHammond , this should help. Let us know if there are any questions.

**Pull request merged**